models/routes.go

```diff
@@ -19,44 +37,65 @@ func (r *Routes) protoRoutes() *ProtoRoutes {
 	return pr
 }
 
-func (r *Routes) Marshal() ([]byte, error) {
-	return r.protoRoutes().Marshal()
-}
-
-func (r *Routes) MarshalTo(data []byte) (n int, err error) {
-	return r.protoRoutes().MarshalTo(data)
-}
-
-func (r *Routes) Unmarshal(data []byte) error {
-	pr := &ProtoRoutes{}
-	err := pr.Unmarshal(data)
-	if err != nil {
-		return err
-	}
-
-	if pr.Routes == nil {
+func (pr *ProtoRoutes) FromProto() *Routes {
+	if pr == nil || pr.Routes == nil || len(pr.Routes) == 0 {
 		return nil
 	}
 
-	routes := map[string]*json.RawMessage{}
+	r := Routes{}
 	for k, v := range pr.Routes {
 		raw := json.RawMessage(v)
-		routes[k] = &raw
+		r[k] = &raw
 	}
-	*r = routes
 
-	return nil
+	return &r
 }
 
-func (r *Routes) Size() int {
-	if r == nil {
-		return 0
-	}
+// func (r *Routes) Marshal() ([]byte, error) {
+// 	return proto.Marshal(r.ToProto())
+// }
 
-	return r.protoRoutes().Size()
-}
+// func (r *Routes) Unmarshal(b []byte) error {
+// 	return proto.Unmarshal(b, r.ToProto())
+// }
+
+// func (r *Routes) MarshalTo(data []byte) (n int, err error) {
+// 	return r.protoRoutes().MarshalTo(data)
+// }
+
+// func (r *Routes) Unmarshal(data []byte) error {
+// 	pr := &ProtoRoutes{}
+// 	err := pr.Unmarshal(data)
+// 	if err != nil {
+// 		return err
+// 	}
+
+// 	if pr.Routes == nil {
+// 		return nil
+// 	}
+
+// 	routes := map[string]*json.RawMessage{}
+// 	for k, v := range pr.Routes {
+// 		raw := json.RawMessage(v)
+// 		routes[k] = &raw
+// 	}
+// 	*r = routes
+
+// 	return nil
+// }
+
+// func (r *Routes) Size() int {
+// 	if r == nil {
+// 		return 0
+// 	}
+
+// 	return r.protoRoutes().Size()
+// }
```

Comment:
delete or uncomment?

Reply:
Cleaned up in [f7314a9](https://github.com/cloudfoundry/bbs/pull/109/commits/f7314a95de15266d952902af8767dd6b5a741df3)